src/test/cli/rbd/not-enough-args.t

```diff
@@ -188,6 +188,7 @@
   rbd: locker was not specified
   [22]
   $ rbd bench-write
+  rbd: bench-write is deprecated, use rbd bench --io-type write ...
```

Comment:
@yangdongsheng So, you still have these deprecated commands in invalid-snap-usage.t and 
 not-enough-args.t tests?